netbox/netbox.py

```diff
@@ -352,27 +425,39 @@ def update_host_meta_vars(self, inventory_dict, host_name, host_vars):
             inventory_dict.update({host_name: host_vars})
         return inventory_dict
 
+    def add_interface_meta_vars(self, current_host, host_vars):
+        """ Adds interfaces information to host_vars dict
+
+        Returns updated host_vars
+        """
+        current_host_id = current_host['id']
+        interfaces = self.get_device_interfaces(current_host_id, self.api_interfaces_url, self.api_token)
+        for interface_id in interfaces:
+            interface = self.get_interface_ip(interface_id, self.api_ipam_url, self.api_token)
+            host_vars.update(interface)
+        return host_vars
+
+
     def generate_inventory(self):
         """Generate Ansible dynamic inventory.
 
         Returns:
             A dict has inventory with hosts and their vars.
         """
-
+https://github.com/KI-labs/netbox-as-ansible-inventory/pull/1/conflict?name=netbox%252Fnetbox.yml&ancestor_oid=ec86312f18771d6d8ab5376e5f67c7e5309f951e&base_oid=dec352823ddab441b63c57d9d79c7e56871102e8&head_oid=90153c9f68aeba7d1854848d55e438552371aa6a
```

Comment:
is this supposed to be here?

Reply:
No. Don't have any idea how it got here. Let me update.

Reply:
Done